src/features/Login/index.jsx

```diff
@@ -1,12 +1,67 @@
-import React, { useState } from 'react';
-import { Form, Input, Button, Checkbox } from 'antd';
-import "./index.scss"
+import React, { useState } from "react";
+import { Form, Input, Button, Row, Col, Space } from "antd";
+import { useHistory } from "react-router-dom"
+import "./index.scss";
+import logo from "../../assets/images/logo.png";
 
 function UserLogin() {
+  let history = useHistory();
+  const redirectRegiter = () => {
+    history.push("/user-register")
 
-    return (
-        <>Trang login</>
-    );
+  }
+
+  return (
+    <>
+      <div className="wrapper_Login">
```

Comment:
Using keyword lowercase for className

Reply:
Fixed at commit [fa21ab90a|https://github.com/NghiaBui486/share-room/pull/1/commits/fa21ab90a7261dcb6a6caf2f0df8eea6157205fd]

Reply:
Fixed at commit [fd36968|https://github.com/NghiaBui486/share-room/pull/1]